README.md

```diff
@@ -146,7 +146,9 @@ ESLint has full support for ECMAScript 3, 5 (default), 2015, 2016, 2017, and 201
 
 ### What about experimental features?
 
-ESLint doesn't natively support experimental ECMAScript language features. You can use [babel-eslint](https://github.com/babel/babel-eslint) to use any option available in Babel.
+ESLint's parser only officially supports the latest final ECMAScript standard. We will make changes to core rules in order to avoid crashes on stage 3 ECMAScript syntax proposals. We may make changes to core rules to better work with language extensions (such as JSX, Flow, and TypeScript) on a case-by-case basis.
+
+In other cases (including if rules need to warn on more or fewer cases due to new syntax, rather than just not crashing), you should use other parsers and/or rule plugins. If you are using Babel, you can use the [babel-eslint](https://github.com/babel/babel-eslint) parser and [eslint-plugin-babel](https://github.com/babel/eslint-plugin-babel) to use any option available in Babel.
```

Comment:
This is very nitpicky, but I prefer `we recommend you use other parsers and/or rule plugins` to `you should use other parsers and/or rule plugins`, because I think it's a little friendlier!

Reply:
Agreed, I'll make that change.

Reply:
Sorry, I lied. Not sure I like that because this really is a (gentle) order to use something that isn't espree. How about `please use other parsers and/or plugins`? Or I can do your wording if you prefer.

Reply:
I prefer the wording I suggested, but I don't feel that strongly about it. Totally up to you :)

Reply:
I don't feel strongly about it, so I'll go with your wording. Expect a new commit shortly.